src/types/accessors/AbstractMetabolicModel.jl

```diff
@@ -155,6 +155,16 @@ negative (2nd law of thermodynamics). This semantic grouping represents ΔGᵣ.
 """
 )
 
+@make_variable_semantics(
+    :environmental_reaction,
+    "Environmental reaction",
+    """
+Community models are composed of member models as well as environmental exchange
+reactions. This semantic grouping represents the environmental exchange
+reactions.
+"""
+)
+
```

Comment:
could we actually call this exchanges right away? :D

Reply:
yeah, but that is even more verbose: `environmental_exchange_reaction`, I wanted to save myself the typing